packages/edge-login-ui-rn/src/util/getHeaderHeight.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import { Platform } from 'react-native'
+
+/**
+ * Calculates the height of the header (where the back button lives).
+ */
+export function getHeaderHeight() {
```

Comment:
Do we really need to cover this area with the dropdown? What if the dropdown doesn't have a minHeight? I know the react-native-airship itself handles the notch thing.

Reply:
Yeah I guess it looks fine without the getHeaderHeight. Thats just the old one I copy so no idea what its really used for. Fixed